src/routes/transactions/mappers/common/transactions.mapper.ts

```diff
@@ -168,15 +173,15 @@ export class TransactionsMapper {
           const transfers = (transaction as EthereumTransaction).transfers;
           if (transfers != null) {
             return await Promise.all(
-              this.mapEthereumTransfer(transfers, chainId, safe),
+              this.mapTransfer(transfers, chainId, safe),
             );
           }
         } else {
           // This should never happen as AJV would not allow an unknown transaction to get to this stage
           throw Error('Unrecognized transaction type');
         }
-      },
+      }),
     );
-    return Promise.all(transactions);
+    return transactions;
```

Comment:
We don't need this function to be `async` we are just returning a promise:

```typescript
private mapGroupTransactions(
    transactionGroup,
    chainId,
    safe,
  ): Promise<TransactionItem[]> {
    return Promise.all(
      transactionGroup.transactions.map(async (transaction) => {
        ...
      }),
    );
```

Reply:
Done 070b3eaec76c5674f35d518d36a85f9af4e888cd